CMakeLists.txt

```diff
@@ -246,6 +247,7 @@ if(OpenCL_ENABLE)
         PATH_SUFFIXES
             "AMD APP/lib/x86_64"
             lib/x86_64
+	    lib/x86_64-linux-gnu
```

Comment:
please fix the indention (spaces only in this file)

Reply:
Updated.